api/db/mongo/mongo.go

```diff
@@ -30,14 +30,6 @@ type DB struct {
 	Session *mgo.Session
 }
 
-// mongoConfig is the struct that represents mongo configuration.
-type mongoConfig struct {
-	Address      string
-	DatabaseName string
-	UserName     string
-	Password     string
-}
```

Comment:
This is a dead code. I removed, by I'm not sure if it is important.

Reply:
That struct is unused probably dead code.